client/components/account-balances/test/index.test.tsx

```diff
@@ -7,26 +7,11 @@ import { render, screen, fireEvent, within } from '@testing-library/react';
 /**
  * Internal dependencies
  */
-import AccountBalances from '../';
-import AccountBalancesHeader from '../header';
-import AccountBalancesTabPanel from '../balances-tab-panel';
-
-import { getGreeting, getCurrencyTabTitle } from '../utils';
-import { useCurrentWpUser } from '../hooks';
+import AccountBalances from '..';
+import { getCurrencyTabTitle } from '../utils';
 import { useAllDepositsOverviews, useInstantDeposit } from 'wcpay/data';
 import { useSelectedCurrency } from 'wcpay/overview/hooks';
 
-const mockUser = {
```

Comment:
Tests related to the AccountOverviewHeader greeting component are not relevant to the AccountBalances component, now exist in `account-overview-header` dir.